dockerfiles/mainnet-1.0.1.env

```diff
@@ -0,0 +1,57 @@
+
+CONTRACT_MCD_FLOP_DEPLOYED=8928163
```

Comment:
I think this might not be the 1.0.1 flop deployment block

Reply:
👍  this was updated in #96